src/colmap/estimators/alignment.cc

```diff
@@ -337,6 +337,65 @@ std::vector<ImageAlignmentError> ComputeImageAlignmentError(
   return errors;
 }
 
+bool AlignReconstructionsViaPoints(const Reconstruction& src_reconstruction,
```

Comment:
It was on my TODO list to add missing unit tests for the other Align* functions in this file. It could be great to add a simple unit test. This could probably synthesize a reconstruction, transform it with a random sim3d and then EXPECT_NEAR the random sim3d.

Reply:
Done.